src/main/webapp/app/exam/manage/students/upload-images/students-upload-images-dialog.component.ts

```diff
@@ -37,8 +37,9 @@ export class StudentsUploadImagesDialogComponent implements OnDestroy {
     notFoundUsers?: NotFoundExamUserType;
     file: File;
 
-    courseId = input.required<number>();
-    exam = input.required<Exam>();
+    // we cannot use input<> here because then the data is undefined
```

Comment:
you can also handle undefined inputs using signals. If they are always undefined however, why do we use inputs?

Reply:
seems to be an incompatibility with dialogs and signals. we will try to come up with a solution in a follow-up